enable_button: 4 # Left trigger button
angular_button: 5 # Right trigger button
frame_mode_button: 6 # Switch to frame movement
joint_mode_button: 7 # Switch to joint movement

axis_linear:
  x: 0
  y: 1
  z: 4

scale_linear:
  x: -0.005
  y: 0.005
  z: 0.005
  
axis_angular:
  x: 0
  y: 1
  z: 4
  
scale_angular:
  x: -0.05
  y: 0.05
  z: 0.05

axis_joints:
  J1: 0
  J2: 1
  J3: 2
  J4: 3
  J5: 4
  J6: 5

scale_joints:
  J1: 0.05
  J2: 0.05
  J3: 0.05
  J4: 0.05
  J5: 0.05
  J6: 0.05

